src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -342,23 +345,155 @@ private LocalDateTime getApplicant1ResponseDeadline(LocalDateTime responseDate,
+                        ));
+
+        sb.append("<h2 class=\"govuk-heading-m\">What happens next</h2>")
+
+            .append("<h3 class=\"govuk-heading-m\">If ")
+            .append(applicantName);
+        if (caseData.getApplicant2() != null) {
+            sb.append(" accept your offer</h3>");
+        } else {
+            sb.append(" accepts your offer</h3>");
+        }
+        sb.append("<ul>")
+            .append("<li>pay ").append(applicantName).append("</li>")
+            .append("<li>make sure any cheques or bank transfers are clear in their account by the deadline</li>")
+            .append("<li>keep proof of any payments you make</li>")
+            .append("</ul>")
+            .append("<p>Contact ")
+            .append(applicantName);
+        if (applicantName.endsWith("s")) {
+            sb.append("'");
+        } else {
+            sb.append("'s");
+        }
+        sb.append(" legal representative if you need details on how to pay.</p>")
+
+            .append("<p>Because you've said you will not pay immediately, ")
+            .append(applicantName)
+            .append(" can either:</p>")
+            .append("<ul>")
+            .append("<li>ask you to sign a settlement agreement to formalise the repayment plan</li>")
+            .append("<li>request a county court judgment against you for &#163;")
+            .append(admitOwed).append("</li>")
+            .append("</ul>")
+
+            .append("<h3 class=\"govuk-heading-m\">If ")
+            .append(applicantName)
+            .append(" disagrees that you only owe &#163;")
+            .append(admitOwed)
+            .append("</h3>")
+            .append("<p>We'll ask if they want to try mediation. ")
+            .append("If they agree we'll contact you to arrange a call with the mediator.</p>")
+            .append(
+                "<p>If they do not want to try mediation the court will review the case for the full amount of &#163;")
+            .append(totalClaimAmount).append(".</p>")
+
+            .append("<h3 class=\"govuk-heading-m\">If ")
+            .append(applicantName)
+            .append(" rejects your offer to pay by ")
+            .append(DateFormatHelper.formatLocalDate(whenWillYouPay, DATE))
+            .append("</h3>")
+            .append("<p>The court will decide how you must pay</p>");
+        return Optional.of(sb.toString());
+    }
+
+    private Optional<String> getPartialAdmitImmediatelySummary(CaseData caseData) {
+        if (!RespondentResponseTypeSpec.PART_ADMISSION.equals(caseData.getRespondent1ClaimResponseTypeForSpec())
+            || !RespondentResponsePartAdmissionPaymentTimeLRspec.IMMEDIATELY.equals(
+            caseData.getDefenceAdmitPartPaymentTimeRouteRequired())) {
+            return Optional.empty();
+        }
+        LocalDate whenWillYouPay = LocalDate.now().plusDays(5);
+        String applicantName = caseData.getApplicant1().getPartyName();
+
+        StringBuilder sb = new StringBuilder();
```

Comment:
Zaki just told me to add a blank line between the green square and the confirmation text, so they are not too near. I'd check that this is not in the same case 
![image](https://user-images.githubusercontent.com/93722198/158798055-5a70712f-e0ae-4f1d-9f22-8949db93b1bc.png)

Reply:
Done in the branch gathering all